tcgithub/tcgithub.go

```diff
@@ -77,12 +77,18 @@ func New(credentials *tcclient.Credentials) *Github {
 	}
 }
 
-// NewFromEnv creates a new client with credentials extract from environment
+// NewFromEnv returns a Github
+// client with credentials taken from the environment variables:
+//
+//  TASKCLUSTER_CLIENT_ID
+//  TASKCLUSTER_ACCESS_TOKEN
+//  TASKCLUSTER_CERTIFICATE
```

Comment:
Would also be worth stating that if `TASKCLUSTER_CLIENT_ID` is not set in environment, *or* is an empty string, that the client will have authentication disabled.